src/set_theory/cardinal.lean

```diff
@@ -375,6 +375,18 @@ instance : comm_semiring cardinal.{u} :=
   npow_zero'    := @power_zero,
   npow_succ'    := λ n c, by rw [nat.cast_succ, power_add, power_one, cardinal.mul_comm] }
 
+@[simp] theorem mul_bit0 (a b : cardinal) : a * (bit0 b) = a * b + a * b :=
+left_distrib a b b
+
+@[simp] theorem mul_bit1 (a b : cardinal) : a * (bit1 b) = a * (bit0 b) + a :=
```

Comment:
Isn't this true in a more general setting?

Reply:
Indeed, we have a `mul_bit0` lemma in the root namespace already

Reply:
Oh! Then I'd only need the exponentiation lemmas, right?

Reply:
Yes - although maybe they should be phrased to match `pow_bit0` (the one about nat powers)